astropy/modeling/powerlaws.py

```diff
@@ -492,6 +491,14 @@ class Schechter1D(Fittable1DModel):
     power-law form of the function cuts off. Note that the default value for
     ``l_star`` is in units of solar luminosities, in erg/s, to avoid overflow
     errors.
+
+    References
+    ----------
+    
+    .. _ref-schechter1976:
```

Comment:
Please use the same numbering format here we use elsewhere, examples are e.g. in the ``functional_models.py`` file.